### PR TITLE
docs: (Platform) added example Menu with icons

### PR DESCRIPTION
**Pull request by olshansky**
#### Please provide a link to the associated issue.
Closes: https://github.com/SAP/fundamental-ngx/issues/3564
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [n/a] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [n/a] update `README.md`
- [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-docs.component.ts [hunk 1]
```diff
@@ -82,5 +86,20 @@ export class PlatformMenuDocsComponent implements OnInit {
...
+        },
+        {
+            language: 'typescript',
+            code: platformWithIconsMenuTsCode,
+            fileName: 'platform-menu-with-icons-example',
+            component: 'PlatformMenuWithIconsExampleComponent'
+        }
+    ];
+
     ngOnInit(): void { }
```
Comment (sKudum) [inline apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-docs.component.ts:104]: Do we need empty function.

File: apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-examples/platform-menu-with-icons-example.component.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+@import "~@angular/cdk/overlay-prebuilt.css";
+@import '~fundamental-styles/dist/menu';
+
+fdp-button {
+    margin-right: 1rem;
+}
+
+fd-image {
```
Comment (sKudum) [inline apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-examples/platform-menu-with-icons-example.component.scss:8]: Query: Is it expected customer should also add this class properties.

File: apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-docs.component.ts (hunk 1 above)
Comment (olshansky) [reply apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-docs.component.ts]: @sKudum yeah, you are right, fixed

File: apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-examples/platform-menu-with-icons-example.component.scss (hunk 2 above)
Comment (olshansky) [reply apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-examples/platform-menu-with-icons-example.component.scss:8]: @sKudum that's just to make it more pretty in example

File: apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-examples/platform-menu-with-icons-example.component.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+@import "~@angular/cdk/overlay-prebuilt.css";
+@import '~fundamental-styles/dist/menu';
```
Comment (Betrozov) [inline apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-examples/platform-menu-with-icons-example.component.scss:2]: I think it is not necessary
It is already defined [here](https://github.com/SAP/fundamental-ngx/blob/90a091104c2e87e50f46117ffd5963072351e068/apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-examples/platform-menu-with-icons-example.component.scss)

File: apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-examples/platform-menu-with-icons-example.component.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+
+@Component({
+    selector: 'fdp-platform-menu-with-icons-example',
+    templateUrl: './platform-menu-with-icons-example.component.html',
+    styleUrls: ['./platform-menu-with-icons-example.component.scss'],
+    encapsulation: ViewEncapsulation.None
+})
+export class PlatformMenuWithIconsExampleComponent {
+    public item = '';
```
Comment (Betrozov) [inline apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-examples/platform-menu-with-icons-example.component.ts:10]: please remove `public`

**Review by Betrozov — CHANGES_REQUESTED** [review]

**Review by Betrozov — APPROVED** [review]

**Review by sKudum — APPROVED** [review]

**Pull request merged**